### PR TITLE
Registration performance history

### PR DESCRIPTION
**Pull request by danwang**

### CODE REVIEW

File: app/assets/javascripts/raxa.js [hunk 1]
```diff
@@ -15,3 +15,80 @@ Raxa.prototype.displayNotice = function(notice, duration) {
...
+
+  var x = d3.time.scale()
+    .domain([d3.min(dates), d3.max(dates)])
+    .range([0, width]);
+
+  var x_axis = d3.svg.axis()
+    .scale(x)
+    .orient("bottom")
+    .ticks(8)
+    .tickFormat(d3.time.format('%b %d'));
```
Comment (seshness) [inline app/assets/javascripts/raxa.js:48]: This is going to need to be param-ified, eventually. Okay for now.

File: app/models/drug.rb [hunk 2]
```diff
@@ -35,11 +42,14 @@ def time_aggregated_data time_period, group_by_period
     return false if not quantities
 
     output = []
-    quantities.group_by do |time, qty|
-      time / group_by_period * group_by_period
+    quantities.group_by do |hash|
```
Comment (seshness) [inline app/models/drug.rb:45]: why?

Comment (seshness) [inline app/models/drug.rb:45]: oh, I see. Never mind.

File: app/views/registrars/show.html.haml [hunk 3]
```diff
@@ -1,10 +1,65 @@
...
+    %td new
+  %tr
+    %td.divider{:colspan => 3}
+      %div
+        -# please change the title attribute of the next line
+        %span
+          %a{:href => '#'} See more
 
-= render :partial => 'registrars/regist_table',
-         :locals => {:registrations_and_divs => @registrations_and_divs}
```
Comment (bucephalus) [inline app/views/registrars/show.html.haml]: I think you need to pull from master?

File: app/models/drug.rb (hunk 2 above)
Comment (seshness) [inline app/models/drug.rb:45]: Please fix related tests next time. I'm taking care of that now. Will append the commit to fix this to this branch/pull request.

File: features/registration/performance_history.feature [hunk 4]
```diff
@@ -12,28 +12,21 @@ Background: Some patients have been registered
       | Patient C |
 
     Given the following registrars exist:
-      | name  |
-      | Reg A |
-      | Reg B |
+      | name  | id |
+      | Reg A | 1  |
```
Comment (seshness) [inline features/registration/performance_history.feature]: why do they have ids?

**Comment by seshness** [comment]
Okay to merge, having looked at the d3-related code. Please don't abstract out the sparkline generation just yet. I'll fix #95 to work with these changes.

**Pull request merged**